packages/utils/types/index.d.ts

```diff
@@ -41,19 +50,19 @@ export interface SerializedStyles {
 export const isBrowser: boolean
 
 export function getRegisteredStyles(
-  registered: RegisteredCache,
+  registered: EmotionRegisteredCache,
   registeredStyles: Array<string>,
   classNames: string
 ): string
 
 export function registerStyles(
-  cache: EmotionCache,
-  serialized: SerializedStyles,
+  cache: EmotionStyleCache,
+  serialized: EmotionSerializedStyles,
   isStringTag: boolean
 ): void
 
 export function insertStyles(
-  cache: EmotionCache,
-  serialized: SerializedStyles,
+  cache: EmotionStyleCache,
+  serialized: EmotionSerializedStyles,
```

Comment:
@Andarist This is what I ended up doing otherwise, they would be incompatible to each other.